.gitlab-ci.yml

```diff
@@ -47,13 +47,15 @@ code_navigation:
 .test:
   # We limit concurrency because the integration tests execute _real_ actions in GitLab
   # By limiting the resource group we can ensure that only one test is run in parallel
-  resource_group: test
   variables:
     GOPATH: "/builds/go"
     GIT_CLONE_PATH: $GOPATH/src/github.com/zaquestion/lab
     LAB_CORE_HOST: "https://gitlab.com"
     LAB_CORE_USER: "lab-testing"
     LAB_CORE_TOKEN: "uDz6czYV412zK-xC5mUu"
+    CI_PROJECT_URL: "https://gitlab.com"
+    GITLAB_USER_LOGIN: "lab-testing"
+    CI_JOB_TOKEN: "uDz6czYV412zK-xC5mUu"
```

Comment:
Why not just set the CI vars to the LAB ones set just before?
```diff
-    CI_PROJECT_URL: "https://gitlab.com"
-    GITLAB_USER_LOGIN: "lab-testing"
-    CI_JOB_TOKEN: "uDz6czYV412zK-xC5mUu"
+    CI_PROJECT_URL: $LAB_CORE_HOST
+    GITLAB_USER_LOGIN: $LAB_CORE_USER
+    CI_JOB_TOKEN: $LAB_CORE_TOKEN
```

Reply:
Oh I thought I had made that change already.  I will fix this right now.

Reply:
you did in the `export` portion in the `script:` section :)

Reply:
Fixed in new version.